<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="ShowCaloriesArrow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CALORIES -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
    </font>
   </property>
   <property name="text">
    <string>By Hayat(V1.0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="MainLabelCalories">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CALORY COUNTER</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShowCalories">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>show calories</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>651</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>90</number>
    </property>
    <property name="rightMargin">
     <number>90</number>
    </property>
    <item row="1" column="2">
     <widget class="QLabel" name="SpeedLAbel">
      <property name="font">
       <font>
        <family>Georgia</family>
       </font>
      </property>
      <property name="text">
       <string>Speed(km/h)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="SlopeSpinBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="TimeLabel">
      <property name="font">
       <font>
        <family>Georgia</family>
       </font>
      </property>
      <property name="text">
       <string>Time(H)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="WeightLAbel">
      <property name="font">
       <font>
        <family>Georgia</family>
       </font>
      </property>
      <property name="text">
       <string>Weight(Kg)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="SlopeLabel">
      <property name="font">
       <font>
        <family>Georgia</family>
       </font>
      </property>
      <property name="text">
       <string>Slope</string>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="WeightSpinBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="TimeSpinBox"/>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="SpeedSpinBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
